build.cmd

```diff
@@ -4,48 +4,21 @@
 :main
 setlocal enabledelayedexpansion
   set errorlevel=
-
-  set lv_api_only=
-  if /I "%~1" == "--api-only" (
-    set lv_api_only=1
-    shift
-  )
-
   set BuildConfiguration=%~1
   if "%BuildConfiguration%"=="" set BuildConfiguration=Debug
 
   set VersionSuffix=%~2
-  if "%VersionSuffix%"=="" set VersionSuffix=alpha-build0000
-
-  set PackageVersion=%~3
-  if "%PackageVersion%"=="" set PackageVersion=1.0.0-%VersionSuffix%
+  if "%VersionSuffix%"=="" set VersionSuffix=beta-build0000
 
   set OutputDirectory=%~dp0LocalPackages
   call :remove_directory "%OutputDirectory%" || exit /b 1
 
   call "%~dp0.\dotnet-install.cmd" || exit /b 1
 
   set procedures=
-
-  if not defined lv_api_only (
-    set procedures=!procedures! build_procdomain
-    set procedures=!procedures! build_xunit_performance_analysis
-    set procedures=!procedures! build_xunit_performance_core
-    set procedures=!procedures! build_xunit_performance_execution
```

Comment:
You should modify the name of your PR to make sure that it's clear that there is a big change here where you no longer build the runner.